src/cpp/session/modules/SessionCopilot.cpp

```diff
@@ -108,7 +108,7 @@ std::map<std::string, std::string> s_extToLanguageIdMap = {
    { ".mjs",   "javascript" },
    { ".ps",    "powershell" },
    { ".py",    "python" },
-   { ".tex",   "latex" },
```

Comment:
Note that this was already set below in https://github.com/rstudio/rstudio/pull/15540/files#diff-5bd55e1a1ba0aa067f6ee707a27f32279b6d6655c9e858cce164d79ea2a6c033R124.